static/styles/left_sidebar.css

```diff
@@ -343,15 +343,18 @@ li.top_left_recent_topics {
     height: 20px;
 }
 
+/* stylelint-disable */
 .conversation-partners,
 .topic-name {
-    display: block;
+    display: -webkit-box;
     width: calc(100% - 5px);
-    white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
     padding-right: 2px;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
```

Comment:
What does this do on non-webkit browsers?  Also, what linter rule are you disabling?

Reply:
@timabbott  I disabled the all `stylelint` rules for these 5 lines of CSS as `display: -webit-box` is not a valid for display property in stylelint library.

and on webkit browers, these CSS ofc won't work if we want to go with CSS only method.
But now a days, Every browser supports -webkit except Internet Explorer

Reply:
There's also an other alternative method to do this change, but it would involve JS - 

Using the Clamp.JS module.

Reply:
Can you post the caniuse details confirming that this will work in all browsers, and test in firefox?  It's important for reviewers to independently verify those details.

Reply:
Hy Tim, I tested this on firefox and it's working perfectly fine!
also here's the ss of `caniuse`

Reply:
![Screenshot from 2022-02-04 17-49-07](https://user-images.githubusercontent.com/76561593/152528287-81fe4c8f-218c-45dc-86c1-867ec45906d4.png)
![Screenshot from 2022-02-04 10-15-08](https://user-images.githubusercontent.com/76561593/152528326-4b5a587b-e385-4b67-9136-2f92b62cc89f.png)